src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search_form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.search_form:before {
+  background: url(images/Search.svg) no-repeat center;
+  background-size: 100%;
+  content: "";
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 50%;
+  transform: translate(0, -50%);
+  position: absolute;
+}
+
+.search_form--sm:before {
+  height: 11px;
+  left: 13px;
+  width: 11px;
+}
+
+.search_field {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  display: block;
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 16px;
+  height: 70px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.4);
+  padding: 0 0 0 62px;
+  width: 100%;
```

Comment:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order